[Source parse ERR] http://www.kff.org
Traceback (most recent call last):
  File "webcrawler.py", line 98, in <module>

  File "webcrawler.py", line 82, in threadpool
    x.next(timeout=5)
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 695, in next
    raise value
  File "/usr/lib/python3.5/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "webcrawler.py", line 74, in go
    NewsSource(source)
  File "webcrawler.py", line 21, in __init__
    self.get_links()
  File "webcrawler.py", line 41, in get_links
    self.url, browser_user_agent=ua.chrome, language='en', config=config)
  File "/usr/local/lib/python3.5/dist-packages/newspaper/api.py", line 30, in build
    s.build()
  File "/usr/local/lib/python3.5/dist-packages/newspaper/source.py", line 88, in build
    self.download()
  File "/usr/local/lib/python3.5/dist-packages/newspaper/source.py", line 177, in download
    self.html = network.get_html(self.url, self.config)
  File "/usr/local/lib/python3.5/dist-packages/newspaper/network.py", line 41, in get_html
    return get_html_2XX_only(url, config, response)
  File "/usr/local/lib/python3.5/dist-packages/newspaper/network.py", line 64, in get_html_2XX_only
    url=url, **get_request_kwargs(timeout, useragent, proxies, headers))
  File "/usr/local/lib/python3.5/dist-packages/requests/api.py", line 72, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/requests/sessions.py", line 502, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.5/dist-packages/requests/sessions.py", line 634, in send
    history = [resp for resp in gen] if allow_redirects else []
  File "/usr/local/lib/python3.5/dist-packages/requests/sessions.py", line 634, in <listcomp>
    history = [resp for resp in gen] if allow_redirects else []
  File "/usr/local/lib/python3.5/dist-packages/requests/sessions.py", line 212, in resolve_redirects
    **adapter_kwargs
  File "/usr/local/lib/python3.5/dist-packages/requests/sessions.py", line 652, in send
    r.content
  File "/usr/local/lib/python3.5/dist-packages/requests/models.py", line 825, in content
    self._content = bytes().join(self.iter_content(CONTENT_CHUNK_SIZE)) or bytes()
  File "/usr/local/lib/python3.5/dist-packages/requests/models.py", line 747, in generate
    for chunk in self.raw.stream(chunk_size, decode_content=True):
  File "/usr/local/lib/python3.5/dist-packages/urllib3/response.py", line 432, in stream
    for line in self.read_chunked(amt, decode_content=decode_content):
  File "/usr/local/lib/python3.5/dist-packages/urllib3/response.py", line 594, in read_chunked
    self._update_chunk_length()
  File "/usr/local/lib/python3.5/dist-packages/urllib3/response.py", line 536, in _update_chunk_length
    line = self._fp.fp.readline()
AttributeError: 'NoneType' object has no attribute 'readline'
[Source parse ERR] http://www.blacklivesmatter.com